src/java.base/share/classes/sun/security/util/DisabledAlgorithmConstraints.java

```diff
@@ -130,21 +130,15 @@ public DisabledAlgorithmConstraints(String propertyName,
         disabledAlgorithms = getAlgorithms(propertyName);
 
         // Check for alias
-        int ecindex = -1, i = 0;
+        Pattern INCLUDE_PATTERN = Pattern.compile(
```

Comment:
You may miss my previous comment.  This variable is not static, hence all capital letters naming does not apply here.

As you are already here, it may be nice if you'd like to have this variable as a static final class field.  Then, the compile() will be called one time most for this class.  As a static class field, you could use the capitalized name.

Reply:
Sorry, I misunderstood what you meant before, I will fix it. Thank you!